packages/components/src/placeholder/index.js

```diff
@@ -15,12 +15,17 @@ import Dashicon from '../dashicon';
 * @param  {Object} props The component props.
 * @return {Object}       The rendered placeholder.
 */
-function Placeholder( { icon, children, label, instructions, className, notices, ...additionalProps } ) {
+function Placeholder( { icon, children, label, instructions, className, notices, mediaPreview, ...additionalProps } ) {
```

Comment:
Is placeholder inherently related to media? By the description of the component a few lines above, it seems to be for use in any context to display an empty state. We shouldn't make this implementation-specific if we don't need to. Could it be fair enough to describe this as merely a preview prop, not "media preview" specifically?

Reply:
It sounds fair indeed. My reasoning for calling it mediaPreview wasn't because of the MediaPlaceholder uses Placeholder, but because generally "media" is previewed. On a second thought a snippet is a preview of text, which isn't media.. so yes, I'll rename it!